controller/composite/controller_revision.go

```diff
@@ -260,7 +260,7 @@ func (pc *parentController) manageRevisions(parent *unstructured.Unstructured, o
 			opts := &metav1.DeleteOptions{
 				Preconditions: &metav1.Preconditions{UID: &revision.UID},
 			}
-			klog.Infof("%v %v/%v: deleting ControllerRevision %v", parent.GetKind(), parent.GetNamespace(), parent.GetName(), revision.GetName())
+			klog.InfoS("Deleting ControllerRevision", "parent_kind", parent.GetKind(), "parent", klog.KObj(parent), "name", revision.GetName())
```

Comment:
I notice few logs starting with Capital case while others start with small case.
What is the appropriate standard?

Reply:
will fix it, just want to confirm if logging format is ok

Reply:
so I think it would be good to follow https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#remove-string-formatting-from-log-message :

> Start from a capital letter.
> Do not end the message with a period.
> Use active voice. Use complete sentences when there is an acting subject ("A could not do B") or omit the subject if the subject would be the program itself ("Could not do B").
> Use past tense ("Could not delete B" instead of "Cannot delete B")
> When referring to an object, state what type of object it is. ("Deleted pod" instead of "Deleted")